shared/src/persistence/dynamo/documents/updateDocketEntryPendingServiceStatus.js

```diff
@@ -6,18 +6,29 @@ exports.updateDocketEntryPendingServiceStatus = async ({
   docketNumber,
   status,
 }) => {
-  await client.update({
-    ExpressionAttributeNames: {
-      '#isPendingService': 'isPendingService',
-    },
-    ExpressionAttributeValues: {
-      ':status': status,
-    },
-    Key: {
-      pk: `case|${docketNumber}`,
-      sk: `docket-entry|${docketEntryId}`,
-    },
-    UpdateExpression: 'SET #isPendingService = :status',
-    applicationContext,
-  });
+  try {
+    await client.update({
```

Comment:
what was the reasoning behind adding all this conditional expression logic?  I wouldn't think our code base should be invoking this method unless this is a real docket entry right?